spec/spec_helper_local.rb

```diff
@@ -57,6 +57,10 @@
   let(:facts) { on_supported_os['redhat-8-x86_64'] }
 end
 
+shared_context 'RedHat 9' do
+  let(:facts) { on_supported_os['redhat-9-x86_64'] }
+end
```

Comment:
I don't think there are facts for this yet. This feels a bit premature. Also, it doesn't look like any test uses this so I'd prefer to leave it out until it's needed.

Reply:
ok